### DISCUSSION TITLE
How to make a custom shape scaling tool

### DISCUSSION THREAD

**Discussion by makleo2002** [clientIO/joint#2635] [Q&A] [answered]
### Introduction

Does anyone have a sample code for creating a shape scaling tool?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by kumilingus** [comment, marked as answer]
Here's *JointJS+* free transform tool: https://resources.jointjs.com/docs/rappid/v4.0/ui.FreeTransform.html
Here's an  element tool example: https://codepen.io/jointjs/pen/poaEpKx
